lunar/distribution.yaml

```diff
@@ -238,11 +238,20 @@ repositories:
       version: kinetic-devel
     status: maintained
   catkin_pip:
+    doc:
+      type: git
+      url: https://github.com/pyros-dev/catkin_pip.git
+      version: devel
     release:
       tags:
         release: release/lunar/{package}/{version}
       url: https://github.com/pyros-dev/catkin_pip-release.git
-      version: 0.2.2-0
+      version: 0.2.3-0
+    source:
+      test_pull_requests: true
```

Comment:
I followed : http://wiki.ros.org/buildfarm/Pull%20request%20testing which was a bit unsettling for me : 
- not sure what "org unit" means
- the first part talking about "org unit" permission has an url : https://github.com/%YOUR_ORG%/**%YOUR_REPO%**/settings/collaboration
And I thought I followed the first point...

I realize I did it only for this repo, so I will manually set the hook up for it now... hang on a bit...